### PR TITLE
US027: Get All Current cars belonging to a Parking Zone

### PR DESCRIPTION
**Pull request by abdullokhtolibjonov**

### CODE REVIEW

File: Parking-Zone/Areas/Admin/Controllers/ParkingZoneController.cs [hunk 1]
```diff
@@ -151,5 +151,21 @@ public IActionResult DeleteConfirmed(Guid id)
...
         }
+
+        public IActionResult GetCurrentCars(Guid zoneId)
+        {
+            var zone = _parkingZoneService.GetById(zoneId);
+
+            if (zone == null)
+                return NotFound("Parking Zone Not Found");
+
+            var plateNumbers = zone.ParkingSlots
```
Comment (DilshodMQ) [inline Parking-Zone/Areas/Admin/Controllers/ParkingZoneController.cs]: Move the logic into Service

**Review by DilshodMQ — CHANGES_REQUESTED** [review]

File: Parking-Zone/Areas/Admin/Controllers/ParkingZoneController.cs [hunk 2]
```diff
@@ -151,5 +151,17 @@ public IActionResult DeleteConfirmed(Guid id)
 
             return RedirectToAction(nameof(Index));
         }
+
+        public IActionResult GetCurrentCars(Guid zoneId)
```
Comment (napster1991) [inline Parking-Zone/Areas/Admin/Controllers/ParkingZoneController.cs]:
```suggestion
        public IActionResult CurrentCars(Guid zoneId)
```

File: Parking-Zone/Services/IParkingZoneService.cs [hunk 3]
```diff
@@ -4,6 +4,6 @@ namespace Parking_Zone.Services
 {
     public interface IParkingZoneService : IService<ParkingZone>
     {
-
+        public List<string> GetPlateNumbers(ParkingZone zone);
```
Comment (napster1991) [inline Parking-Zone/Services/IParkingZoneService.cs]:
```suggestion
        public List<string> GetCurrentCarsPlateNumbersByZone(ParkingZone zone);
```

**Review by napster1991 — CHANGES_REQUESTED** [review]
Where are tests for ParkingZoneService.GetPlateNumbers() ?

**Review by napster1991 — APPROVED** [review]

**Comment by abdullokhtolibjonov** [comment]
[US027: Get all current cars (plate numbers) in the parking zone. The zone card (on /Zones/Index) must have a “Current cars” link. Add a corresponding page that lists all current cars in the zone.](https://trello.com/c/pA0ItdI9/36-us027-get-all-current-cars-plate-numbers-in-the-parking-zone-the-zone-card-on-zones-index-must-have-a-current-cars-link-add-a-co)

**Pull request merged**